packages/react/src/components/header/components/headerActionBarItem/HeaderActionBarItemButton.module.scss

```diff
@@ -0,0 +1,103 @@
+@use '../../../../styles/common.scss';
+
+button.actionBarItemButton {
+  align-self: center;
+  appearance: none;
+  background: transparent;
+  border: none;
+  color: var(--header-color);
+  cursor: pointer;
+  padding: 10px;
+
+  &:focus-visible {
+    border: none;
+    outline: var(--header-focus-outline-width) solid var(--header-focus-outline-color);
+    outline-offset: var(--header-focus-outline-width);
+  }
+
+  &.fullWidth,
+  &:focus-visible {
+    outline-offset: calc(-1 * var(--header-focus-outline-width)) !important;
+  }
+
+  &.fullWidth {
+    border: 2px solid var(--header-color);
+    display: flex;
+    gap: 8px;
+    justify-content: center;
+    line-height: 24px;
+    margin: 18px;
+    padding: 16px 24px;
+
```

Comment:
Variables for all these spacings: gap, line-height, margin, padding?

Reply:
fixed